magner/lib/views/components/page-header.vue

```diff
@@ -1,8 +1,14 @@
 <template>
   <div v-if="header.title || tabs.tabs.length" class="magner-page-header">
-    <h1 class="magner-page-header_title">
-      <template v-if="header.title">{{ customT(header.title) }}</template>
-    </h1>
+    <div class="magner-page-header_title-container">
+      <h1 class="magner-page-header_title">
+        <template v-if="header.title">{{ customT(header.title) }}</template>
+      </h1>
+
+      <span v-if="showFiltersBtn" class="magner-page-header_filters-btn" @click="toggleFilters">
+        <svg-icon core="filter" />
+      </span>
```

Comment:
Question: is it OK to put the filters button here? As I can see, `showFiltersBtn` is added into the store. `<PageHeader>` is also used in the card page, but why would a user need filters there?

Reply:
And an important remark: please, use `<button>` element to preserve HTML-semantics. Clickable `span`s are not accessible by many devices. Even better if you could use Element-plus's button for icons.